host.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
can be removed. was a way that i tested, to loadjson through a azure function method. Since we aren't using it, we can remove this